[2025-04-22T20:28:38.022+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-22T20:30:51.534+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_files_projetos_grupo_1.upload_file_projeto_2410_belov scheduled__2025-04-21T17:00:00+00:00 [queued]>
[2025-04-22T20:30:51.540+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_files_projetos_grupo_1.upload_file_projeto_2410_belov scheduled__2025-04-21T17:00:00+00:00 [queued]>
[2025-04-22T20:30:51.541+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-22T20:30:51.552+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_file_projeto_2410_belov> on 2025-04-21 17:00:00+00:00
[2025-04-22T20:30:51.556+0000] {standard_task_runner.py:72} INFO - Started process 263 to run task
[2025-04-22T20:30:51.559+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'upload_files_projetos_grupo_1', 'upload_file_projeto_2410_belov', 'scheduled__2025-04-21T17:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/dag_projeto_grupo-01.py', '--cfg-path', '/tmp/tmpk3l8y_yl']
[2025-04-22T20:30:51.561+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask upload_file_projeto_2410_belov
[2025-04-22T20:30:51.596+0000] {task_command.py:467} INFO - Running <TaskInstance: upload_files_projetos_grupo_1.upload_file_projeto_2410_belov scheduled__2025-04-21T17:00:00+00:00 [running]> on host e50052774237
[2025-04-22T20:30:51.645+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='arthur.lopes' AIRFLOW_CTX_DAG_ID='upload_files_projetos_grupo_1' AIRFLOW_CTX_TASK_ID='upload_file_projeto_2410_belov' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-21T17:00:00+00:00'
[2025-04-22T20:30:51.646+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-22T20:28:38.387+0000] {_universal.py:506} INFO - Request URL: 'https://rpeottaprojeto.blob.core.windows.net/belov-2410/LD-2410-GR-PLN-GRC-001_R7.xlsx'
Request method: 'PUT'
Request headers:
    'Content-Length': '134259'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.10.16 (Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': '5f6c5cea-1fb8-11f0-abc0-9e5d745be2f1'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-04-22T20:28:40.053+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Tue, 22 Apr 2025 20:28:39 GMT'
    'ETag': '"0x8DD81DC448D08AC"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '6b60b8f2-601e-002e-0cc5-b340dc000000'
    'x-ms-client-request-id': '5f6c5cea-1fb8-11f0-abc0-9e5d745be2f1'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Tue, 22 Apr 2025 20:28:39 GMT'
[2025-04-22T20:28:40.056+0000] {logging_mixin.py:190} INFO - Arquivo 'LD-2410-GR-PLN-GRC-001_R7.xlsx' enviado com sucesso para o container 'belov-2410'.
[2025-04-22T20:28:40.444+0000] {_universal.py:506} INFO - Request URL: 'https://rpeottaprojeto.blob.core.windows.net/belov-2410/CS-2410-GR-PLN-GER-001%3DR2.xlsx'
Request method: 'PUT'
Request headers:
    'Content-Length': '66468'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.10.16 (Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': '60a643c8-1fb8-11f0-abc0-9e5d745be2f1'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-04-22T20:28:41.789+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Tue, 22 Apr 2025 20:28:41 GMT'
    'ETag': '"0x8DD81DC45908349"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '87c11133-801e-0036-24c5-b39fbb000000'
    'x-ms-client-request-id': '60a643c8-1fb8-11f0-abc0-9e5d745be2f1'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Tue, 22 Apr 2025 20:28:41 GMT'
[2025-04-22T20:28:41.792+0000] {logging_mixin.py:190} INFO - Arquivo 'CS-2410-GR-PLN-GER-001=R2.xlsx' enviado com sucesso para o container 'belov-2410'.
[2025-04-22T20:28:41.909+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/dag_projeto_grupo-01.py", line 228, in upload_file_belov_2410
    upload_to_blob(
  File "/opt/airflow/dags/dag_projeto_grupo-01.py", line 24, in upload_to_blob
    raise e
  File "/opt/airflow/dags/dag_projeto_grupo-01.py", line 18, in upload_to_blob
    with open(local_file_path,"rb") as data:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/dados/02. PLN_Operacional/01 - PLANEJAMENTO E CONTROLE/COORDENACAO/CONTROLES (POWER BI)/240124/Receita 23-24/PNL MASTER _TELMO_2024/PLN MASTER  FATURAMENTO ANUAL 24_R4.xlsx'
[2025-04-22T20:28:41.928+0000] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=upload_files_projetos_grupo_1, task_id=upload_file_projeto_2410_belov, run_id=scheduled__2025-04-21T17:00:00+00:00, execution_date=20250421T170000, start_date=20250422T203051, end_date=20250422T202841
[2025-04-22T20:28:41.949+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-22T20:28:41.950+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 35 for task upload_file_projeto_2410_belov ([Errno 2] No such file or directory: '/opt/airflow/dados/02. PLN_Operacional/01 - PLANEJAMENTO E CONTROLE/COORDENACAO/CONTROLES (POWER BI)/240124/Receita 23-24/PNL MASTER _TELMO_2024/PLN MASTER  FATURAMENTO ANUAL 24_R4.xlsx'; 263)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/dag_projeto_grupo-01.py", line 228, in upload_file_belov_2410
    upload_to_blob(
  File "/opt/airflow/dags/dag_projeto_grupo-01.py", line 24, in upload_to_blob
    raise e
  File "/opt/airflow/dags/dag_projeto_grupo-01.py", line 18, in upload_to_blob
    with open(local_file_path,"rb") as data:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/dados/02. PLN_Operacional/01 - PLANEJAMENTO E CONTROLE/COORDENACAO/CONTROLES (POWER BI)/240124/Receita 23-24/PNL MASTER _TELMO_2024/PLN MASTER  FATURAMENTO ANUAL 24_R4.xlsx'
[2025-04-22T20:28:41.959+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-22T20:28:41.970+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-22T20:28:41.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
